config/element_io_develop.json

```diff
@@ -0,0 +1,13 @@
+{
+  "posthog": {
+    "api_key": "phc_MhClVy9DiV20vazSYIiedFkM5Xi3z1LPBwrdn9PYZQQ",
```

Comment:
This means the keys are just fully public on gh?
Which is not a big deal since they can be read inside the a browser terminal anyways?

Reply:
Yeah, this is what we do for element-web: https://github.com/vector-im/element-web/blob/develop/element.io/app/config.json#L49